test/unit_LinearMap.jl

```diff
@@ -83,6 +83,9 @@ for N in [Float64, Rational{Int}, Float32]
     M = zeros(N, 2, 2)
     vlist = vertices_list(LinearMap(M, b))
     @test vlist == [zeros(N, 2)]
+    X = rand(Interval) × ZeroSet(1)
```

Comment:
In principle you'd use `rand(Interval, N=N)` to create a random interval of numeric type `N` (such that it matches the numeric type of `M` -- because the constructor of `M * X` doesn't handle different numeric types, although being supported by the lazy linear map).

However, `rand(Interval, N=Rational{Int})` doesn't work. So i would suggest to pick any interval, say `X = Interval(N(0), N(1))`.